tests/robot-tests/tests/admin_and_public/bau/publish_release_and_amend.robot

```diff
@@ -81,17 +81,17 @@ Create data block table
 
     user waits until page contains   Dates test subject
     user clicks radio    Dates test subject
-    user clicks element   css:#publicationSubjectForm-submit
+    user clicks element   id:publicationSubjectForm-submit
 
```

Comment:
Not sure if these empty spaces throughout the PR are on purpose (in order to make the test cases more readable I assume) but if they aren't then they need to be removed. If they are on purpose then everything looks good and it can be merged